docs/how-to-guides/running-autoware-without-cuda.md

```diff
@@ -0,0 +1,29 @@
+# Running Autoware without CUDA
+
+Two following algorithms in Autoware depend on a CUDA environment.
```

Comment:
Since Autoware Core won't depend on CUDA, I think it's good to say Autoware Universe here.

```suggestion
Two following algorithms in Autoware Universe depend on a CUDA environment.
```

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/128/commits/ad7a25824788219877f9b9ffde001e6e0520ab41
Fixed! Thank you for the suggestion